src/content/get-started/dev-quickstart.mdx

```diff
@@ -0,0 +1,92 @@
+3. From the Okteto Catalog, select the application you want to develop. Okteto will begin deploying the chosen application.
+4. Once the application is deployed, your dev environment is ready. Okteto will create endpoints for all the microservices and external resources (if configured by the platform team) as part of your application and give you the links view them.
+
+## Step 4: Connect to Your Okteto Instance
+
+1. If you haven't already, [install the Okteto CLI following the guide here](get-started/install-okteto-cli.mdx).
+2. Open a terminal and point the Okteto CLI to your Okteto instance by running:
+
+   ```bash
+   okteto context
+   ```
+
+   And then typing in the URL for your Okteto instance when prompted.
+
+3. You’ll be taken to the browser and asked to authenticate. Once that is done, Okteto CLI is configured and ready to be used with your development environments.
+
+## Step 5: Start Your Development Container
+
+1. Clone the repository of the application you just deployed to your local machine.
+2. Open the cloned repository in your favorite code editor (Okteto supports any code editor and other tools you prefer).
+3. In the terminal, navigate to the application’s directory.
+4. Start the development container by running:
+
+   ```bash
+   okteto up
+   ```
+
+5. You will be prompted to select the microservice you want to work on. Choose the appropriate service to proceed.
+6. If you need to work on multiple services simultaneously, you can launch another `okteto up` session in a different terminal window.
+
+## Step 6: Begin Development
+
+1. Once the development container is up, you will have terminal access inside of the dev container.
+2. Start the development process by running the relevant command for your application. For example, for a React application, run:
+
+   ```bash
+   yarn start
+   ```
+
+## Step 7: Sync Code and View Changes
+
+- The dev container syncs all your local code changes with the code running on the cloud cluster.
+- Any changes you make locally will be reflected in the endpoints shown in the Okteto UI, allowing you to see your changes in a production-like environment instantly.
+
+## Benefits of Using Okteto
+
+- **No Configuration Required**: You can start writing code immediately using the preconfigured applications in the Okteto catalog.
+- **Resource Efficiency**: You are not limited by your local machine’s resources when running resource-intensive services and applications.
+- **Immediate Feedback**: You can see your changes running in a production-like environment as soon as you write code, without waiting for CI, staging, or test environments.
+
+## Next Steps
+
+Dive into learning about Okteto by our [CLI and Dashboard here](/get-started/using-okteto-cli-and-dashboard.mdx).
+
+Happy hacking!
```

Comment:
In all other tutorials, we are using `Happy coding!`. Happy to change it, but we should be consistent across docs